package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "global-fishing-watch",
-  "version": "2.1.0",
+  "version": "2.4.0",
   "description": "This repository will host the Global Fishing Watch client application developed by Vizzuality",
   "main": "server.js",
   "scripts": {
     "analyze-bundle": "webpack --config ./config/webpack.config.js --json | webpack-bundle-size-analyzer",
-    "build": "webpack --config ./config/webpack.config.js --progress --profile --colors",
+    "build": "NODE_ENV='production' webpack --config ./config/webpack.config.js --progress --profile --colors",
```

Comment:
When deploying this, we run `NODE_EN=-production npm run build`. Shouldn't this variable be inherited by the webpack process?

Reply:
Both work, but in reality AFAIK `build` is never called with anything else than 'production', considering 'development' is just for local testing purposes, and the local environment doesn't use `npm run build` but the webpack dev server.
I'm happy to remove that edit if you prefer.